README.md

```diff
@@ -64,6 +64,48 @@ However, processing all the datasets will take a long time and memory.
 
 *Note*: `poetry run` runs commands from within the virtual environment. You can also activate it with `poetry shell` and then simply run `etl ...`.
 
+#### Creating the pipeline of a new dataset
+
+These are the steps to create a data pipeline for a dataset called `example_dataset`, from an institution called
+`example_institution`, with version `YYYY-MM-DD` (where this date tag can typically be the current date when the dataset
+is being added to `etl`, or the date when the source data was released or updated):
+
+0. Activate the virtual environment (running `poetry shell`).
+1. **Create a new branch in the `walden` submodule**.
+2. **Create an ingest script** (e.g. `etl/vendor/walden/ingests/example_institution.py`) to download the data from its
+original source and upload it as a new data snapshot into the S3 `walden` bucket.
```

Comment:
I think it would be good to have a bit more information on this. For example, you need to have your AWS config set up. What would a manual upload look like in practice (maybe too much for this README but I think it should be documented somewhere. Maybe it is also worth having an example ingest script - or link to an existing one?

Reply:
Yes for the example script. I would not even mention the manual upload as I think we should never do this.